source/Public/New-GuestConfigurationPolicy.ps1

```diff
@@ -31,6 +31,25 @@
         The public HTTP or HTTPS URI of the Guest Configuration package (.zip) to run via the created policy.
         Example: https://github.com/azure/auditservice/release/AuditService.zip
 
+        Note: If you are using an Azure storage account to store the custom machine configuration package artifact, you have two options for access:
+        1. Generate a blob shared access signature (SAS) token with read access and provide the full blob URI with the SAS token for the ContentUri parameter.
+        2. Create a user-assigned managed identity with read access to the storage account blob containing the package.
+            Provide the resource ID of the managed identity, a local path to the zipped package, and a URI to the package without a SAS token for the ManagedIdentityResourceId, LocalContentPath, and ContentUri parameters.
+            With this option, once the generated policy is applied, the managed identity will be used to download the package onto the target machine.
+
+    .PARAMETER ManagedIdentityResourceId
+        This is the identity that is used to download the package from storage account container instead of using SaS url.
+        The value for this parameter needs to be the resource id of the managed identity.
+        This is an option to use when the package is stored in a storage account and the storage account is protected by a managed identity.
+
+        Note: optional parameter. If this is specified, LocalContentPath must also be specified.
+
+    .PARAMETER LocalContentPath
```

Comment:
Should the customer be allowed to specify this parameter without ManagedIdentity?

Reply:
Yes, so that day we decided on including it irrespective of the newer workflow.